disco/types/message.py

```diff
@@ -509,6 +509,14 @@ def delete_reaction(self, emoji, user=None):
             emoji,
             user)
 
+    def delete_all_reactions(self):
+        """
+        Deletes all the reactions from a message.
+        """
+        self.client.api.channels_messages_reactions_delete_all(
+            self.channel_id,
+            self.id)
```

Comment:
minor nit; line break the paren and add a trailing comma

```suggestion
            self.id,
        )
```